app/controllers/chargeback_controller.rb

```diff
@@ -747,10 +747,17 @@ def get_tags_all(category)
     classification.entries.each { |e| @edit[:cb_assign][:tags][e.id.to_s] = e.description } if classification
   end
 
+  DEFAULT_CHARGEBACK_LABELS = ["com.redhat.component"]
```

Comment:
This should be seeded from a file but im really not sure whats the correct way :sweat:

Reply:
Yeah.. Let's wait for @himdel to respond on this..

Reply:
If this is just one constant, a file would probably be an overkill, this is fine :+1: (at least until it becomes some structured data, then we can deal with creating a yaml and importing during seed).